ext/OceananigansPythonCallExt.jl

```diff
@@ -0,0 +1,146 @@
+module OceananigansPythonCallExt
+
+using PythonCall
+using CondaPkg
+using SparseArrays
+
+using Oceananigans
+using Oceananigans.Grids: λnodes, φnodes, Center, Face
+using Oceananigans.Architectures: on_architecture, CPU
+
+import Oceananigans.Fields: regridding_weights
+
+"""
+    add_package(package_name, channel="conda-forge"; verbose=true)
+
+Install `package_name` with `CondaPkg.add` from `channel`, printing
+a few messages if `verbose == true`.
+Return a NamedTuple containing package information if successful.
+"""
+function add_package(name, channel="conda-forge"; verbose=true)
+    verbose && @info "Installing $(name)..."
+    CondaPkg.add(name; channel)
+    pkg = CondaPkg.which(name)
+    verbose && @info "... $name has been installed at $(pkg)."
+    return pkg
+end
+
+"""
+    add_import_pkg(package_name, channel="conda-forge")
+
+Import and return `package_name` with `PythonCall.pyimport`,
+installing it with `add_package` if it is not found.
+"""
+function add_import_pkg(name, channel="conda-forge")
```

Comment:
This isn't how we install Conda packages inside Julia packages - declare them in a CondaPkg.toml file instead.